knowledge-base/grid-state-filter-nullable-bool.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Programatically filter a nullable bool column in the Grid by null values
```

Comment:
Use title case [capitalization](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).